path_roles.go

```diff
@@ -345,13 +345,18 @@ func (b *azureSecretBackend) createPersistedApp(ctx context.Context, req *logica
 		return err
 	}
 
+	assignmentIDs, err := c.generateUUIDs(len(role.AzureRoles))
+	if err != nil {
+		return fmt.Errorf("error generating assginment IDs; err=%w", err)
```

Comment:
Kind of a nit, but:
- Assignment is misspelled
- For consistency, I'd suggest to follow the general error format found throughout the secrets engine similar to `fmt.Errorf("error description: %w", err)`